python-sdk/src/astro/databases/mssql.py

```diff
@@ -392,6 +392,34 @@ def _build_merge_sql(
         }
         return statement, params
 
```

Comment:
Do we have any tests covering the changes?

Reply:
@tatiana We have it here - [python-sdk/tests_integration/sql/test_table.py](https://github.com/astronomer/astro-sdk/pull/1752/files#diff-37236c13591e7bd7f0ee1c26f75fe36b4a631141b28fe3e81f35a16fd0a1362b)